salt/orchestrate/aws/cloud_profiles/backup_host.conf

```diff
@@ -2,6 +2,7 @@ backup_host:
   provider: mitx
   size: t2.medium
   image: ami-49e5cb5e
+  availability_zone: us-east-1b
```

Comment:
I think I had put it there because the EBS volume that we're attaching has to be in the same AZ as the host.